### PR TITLE
Add dualstack periodic job

### PR DESCRIPTION
**Pull request by andfasano**
This PR adds a periodic job to execute the `e2e-metal-ipi` test with the dualstack configuration enabled. See https://github.com/openshift-metal3/dev-scripts/pull/1050

### CODE REVIEW

File: ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master.yaml [hunk 1]
```diff
@@ -41,6 +41,16 @@ tests:
     - ref: baremetalds-devscripts-conf-dualstack
     - ref: baremetalds-devscripts-setup
     workflow: baremetalds-e2e
+- as: e2e-metal-ipi-dualstack-periodic
+  cron: 0 */8 * * *
```
Comment (danwinship) [inline ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master.yaml:45]: `*/8` means 'every 8 hours' I guess? Maybe just make it once a day for now and we can update it later

Comment (andfasano) [reply ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master.yaml]: Done

**Pull request merged**